<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML>
<HEAD>
	<META HTTP-EQUIV="CONTENT-TYPE" CONTENT="text/html; charset=iso-8859-1">
	<TITLE>Completing the Expression Catalog of the Arabidopsis Transcript</TITLE>
	<META NAME="GENERATOR" CONTENT="OpenOffice.org 2.0  (Linux)">
	<META NAME="AUTHOR" CONTENT="Julia Redman">
	<META NAME="CREATED" CONTENT="20070223;11410000">
	<META NAME="CHANGEDBY" CONTENT="Julia Redman">
	<META NAME="CHANGED" CONTENT="20070223;11410000">
	<STYLE>
	<!--
		@page { size: 8.5in 11in; margin-right: 1in; margin-top: 1in; margin-bottom: 0.5in }
		P { margin-bottom: 0.08in; direction: ltr; color: #000000; widows: 2; orphans: 2 }
		P.western { font-family: "Times New Roman", serif; font-size: 12pt; so-language: en-US }
		P.cjk { font-family: "Times New Roman", serif; font-size: 12pt }
		P.ctl { font-family: "Times New Roman", serif; font-size: 12pt; so-language: ar-SA }
		A:link { color: #0000ff }
	-->
	</STYLE>
</HEAD>
<BODY LANG="en-US" TEXT="#000000" LINK="#0000ff" DIR="LTR">
<P CLASS="western" STYLE="margin-bottom: 0in"><FONT SIZE=4 STYLE="font-size: 16pt"><B>Completing
the Expression Catalog of the Arabidopsis Transcriptome by
Quantitative Real-Time RT-PCR</B></FONT></P>
<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
</P>
<P CLASS="western" STYLE="margin-bottom: 0in"><B>QPCR SOP</B></P>
<P CLASS="western" STYLE="margin-bottom: 0.17in"><BR><BR>
</P>
<P CLASS="western" STYLE="margin-bottom: 0.17in">TABLE OF CONTENTS:</P>
<P CLASS="western" STYLE="margin-bottom: 0.17in">SECTION 1: PLANT RNA
EXTRACTION	PAGE 2</P>
<P CLASS="western" STYLE="margin-bottom: 0.17in">SECTION 2: RNA
PROCESSING	PAGE 4</P>
<P CLASS="western" STYLE="margin-bottom: 0.17in">SECTION 3: cDNA
SYNTHESIS FOR RT-qPCR	PAGE 6</P>
<P CLASS="western" STYLE="margin-bottom: 0.17in">SECTION 4: cDNA QC
TESTING	PAGE 8</P>
<P CLASS="western" STYLE="margin-bottom: 0.17in">SECTION 5: qPCR
REACTION SETUP	PAGE 10</P>
<P CLASS="western" STYLE="margin-bottom: 0.17in"><BR><BR>
</P>
<P CLASS="western" STYLE="margin-bottom: 0.17in; page-break-before: always">
<a name="sec1"></a>
<B>SECTION 1:  PLANT RNA EXTRACTION</B><BR><BR>NOTES BEFORE
BEGINNING:<BR><BR>--For most tissues/treatments, 2g of tissue will
produce plenty of RNA for our needs.<BR><BR>--ALWAYS use blocked
tips, clean pipettes, and gloved hands when working with RNA. If you
wear a lab coat, DO NOT use the same one used for activities like
seed harvesting or bacterial mini-preps.<BR><BR>--Day 1 steps using
phenol and chloroform should be carried out in the fume hood.<BR><BR><BR>
</P>
<P CLASS="western" STYLE="margin-bottom: 0in; border-top: 1px solid #000000; border-bottom: none; border-left: none; border-right: none; padding-top: 0.01in; padding-bottom: 0in; padding-left: 0in; padding-right: 0in">
<B>Reagents and Supplies</B></P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	RNase free tubes 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">Mortar and pestle 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">liquid Nitrogen 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">5:1 Acid
	Phenol:Chloroform (Ambion p/n 9722) 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">Chloroform:IAA 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">DEPC-treated H2O
	(Ambion p/n 9920) 
	</P>
	<LI><P LANG="it-IT" CLASS="western" STYLE="margin-bottom: 0in">8M
	LiCl (Sigma p/n L-7026) 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">2M LiCl, 4C 
		</P>
	</UL>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">80% EtOH, -20C 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in"><B>Extraction
	Buffer</B>, made fresh each time (Per 10mL needed) 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">50mM Tris pH 7.5
		(0.5mL 1M) 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">100mM LiCl
		(0.125mL 8M) 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">5mM EDTA, pH8
		(0.1mL 0.5M) 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">1% SDS (1mL 10% )
				</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">1%
		(beta)-Mercaptoethanol (0.1mL) 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">DEPC-treated
		H<SUB>2</SUB>O (8.2mL)</P>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in; border-top: 1px solid #000000; border-bottom: none; border-left: none; border-right: none; padding-top: 0.01in; padding-bottom: 0in; padding-left: 0in; padding-right: 0in">
<BR>
</P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	<B>Day 1</B> 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Grind 1g of
		tissue in mortar and pestle under liquid nitrogen, adding sand if
		necessary, to fine powder. Transfer to tube in liquid nitrogen. 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">While the
			protocol may be scaled down, scaling up above 1g per tube is not
			recommended. Instead, make multiple preps when more RNA is needed.
						</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in"><I>Tissue
			preparation may be done in advance, storing the tissue at -80C and
			keeping under liquid nitrogen until added to buffer</I> 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Dump tissue into
		10mL Extraction buffer. Cap and vortex briefly to break clumps. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Quickly add 10mL
		5:1 Acid Phenol:Chloroform. Vortex 1min. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Centrifuge
		4000rpm, 10min at RT in table top centrifuge. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Recover aqueous
		phase, avoiding any floating particles and the interface. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Add 10mL
		Chloroform:IAA (using glass pipette) and vortex or shake
		vigorously. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Centrifuge
		4000rpm, 10min at RT. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Recover ~9mL
		aqueous phase. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Add 1/3 volume 8M
		LiCl (3mL) and vortex. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Incubate at
		least overnight at -20C. 
		</P>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
</P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	<B>Day 2</B> 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Remove tubes from
		-20C and let thaw briefly. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Centrifuge
		4000rpm, 10min at 4C. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Remove
		supernatant and SAVE pellet. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Resuspend pellet
		in 3mL DEPC-treated H<SUB>2</SUB>O. 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in"><I>If there are
			duplicate tubes, they may be combined at this point</I> 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Add 1/3 vol 8M
		LiCl and vortex. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Incubate at
		least overnight at -20C. 
		</P>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
</P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	<B>Day 3</B> 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Remove tubes from
		-20C and let thaw briefly. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Centrifuge
		4000rpm, 10min at 4C. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Decant
		supernatant. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Wash pellet with
		4C 2M LiCl. 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Add 4mL and
			vortex. 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Centrifuge
			4000rpm, 5min at 4C. 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Decant
			supernatant. 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Repeat wash once.
				</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Wash pellet with
		-20C 80% EtOH. 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Add 4mL and
			vortex. 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Centrifuge
			4000rpm, 5min at 4C. 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Decant
			supernatant. 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Repeat wash once.
				</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Invert tube and
		air dry pellet no more than 15min. Over dried RNA is hard to
		dissolve. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Resuspend pellet
		in 200ul DEPC-treated H<SUB>2</SUB>O per 1g starting material. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Check yield. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Store at -80C,
		aqueous. 
		</P>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in; page-break-before: always">
<a name="sec2"></a>
<B>SECTION 2:  RNA PROCESSING</B></P>
<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
</P>
<P CLASS="western" STYLE="margin-bottom: 0.17in">NOTES BEFORE
BEGINNING:<BR><BR>--ALWAYS use blocked tips, clean pipettes, and
gloved hands when working with RNA. If you wear a lab coat, DO NOT
use the same one used for activities like seed harvesting or
bacterial mini-preps.<BR><BR><BR>
</P>
<P CLASS="western" STYLE="margin-bottom: 0in; border-top: 1px solid #000000; border-bottom: none; border-left: none; border-right: none; padding-top: 0.01in; padding-bottom: 0in; padding-left: 0in; padding-right: 0in">
<B>Supplies</B></P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	RNase free tubes 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">TURBO DNA-free&trade;
	(Ambion p/n AM1907) 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">RNeasy Mini Kit
	(TIGR stock room # 5802) 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">Mini-Gel box and
	accessories 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">Gel Reagents 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">1M NaPO<SUB>4 </SUB>pH
		6.5 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Agarose 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Formaldehyde 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Formamide 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">0.5M EDTA pH 8.0 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Ethidium Bromide 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">6x loading dye 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">RNA Millenium
		Markers (Ambion p/n AM7150)</P>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in; border-top: 1px solid #000000; border-bottom: none; border-left: none; border-right: none; padding-top: 0.01in; padding-bottom: 0in; padding-left: 0in; padding-right: 0in">
<BR>
</P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	<B>DNase Treatment</B> 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Aliquot 300ug
		Total RNA 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Add DEPC-treated
		H<SUB>2</SUB>O to 267ul, 30ul 10x DNase buffer, and 3ul DNAse I;
		flick to mix. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Incubate 30min at
		37C. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Move directly
		to filtration. 
		</P>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
</P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	<B>RNeasy Filtration</B> 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in"><I>Apply about
		50ug RNA to each column for best results</I> 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in"><SPAN LANG="it-IT">Add
		300ul DEPC-treated H<SUB>2</SUB>O, aliquot 100ul per tube. </SPAN>
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Continue with the
		QIAGEN protocol for RNA cleanup as stated in the book, with the
		following notes in mind: 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Pass the sample
			over the column twice. 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Include optional
			step 6a. 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Elute in 50ul
			DEPC-treated H<SUB>2</SUB>O, allowing water to sit on column 10
			min prior to centrifuging. 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Combine like
		samples. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Check yield. 
		</P>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
</P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	<B>RNA Denaturing Gel Analysis</B> 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in"><I>Make gel and
		buffer fresh each time, excess sample buffer may be kept up to 2
		months at 4C</I> 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Gel Preparation
		(appropriate amount for mini-gel) 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">44mL dH<SUB>2</SUB>O
						</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">1mL 1M NaPO<SUB>4</SUB>
			pH 6.5 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">0.5g Agarose 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Microwave to
			melt, cool to hand-hot. 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Add 4mL
			Formaldehyde, mix and pour. 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Running Buffer 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">20mL
			Formaldehyde 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">2.5mL 1M NaPO<SUB>4</SUB>
			pH 6.5 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">dH2O to 250mL 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">2x Sample Buffer 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">50ul Formamide 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">27.5ul
			DEPC-treated H<SUB>2</SUB>O 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">16.5ul
			Formaldehyde 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">2ul 0.5M EDTA 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">2ul Ethidium
			Bromide 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Sample
		preparation 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Mix RNA (usually
			1ul is appropriate) with DEPC-treated H<SUB>2</SUB>O to 5ul. 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Add 5ul 2x
			Sample Buffer 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Incubate at 65C
			for 15min 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Quick cool. 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Add 2ul loading
			dye. 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Load and Run gel 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">80V for
			45-60min. 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in"><I>Running gel
			longer than 60min can lead to poor results.</I> 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Results
			should show 2 or 3 clean bands, depending on tissue type. 
			</P>
		</UL>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in; page-break-before: always">
<a name="sec3"></a>
<B>SECTION 3:  cDNA SYNTHESIS FOR RT-qPCR</B></P>
<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
</P>
<P CLASS="western" STYLE="margin-bottom: 0.17in">&gt;&gt;Batch
synthesis with SuperScript III, modified from Invitrogen
protocol&lt;&lt;<BR><BR>NOTES BEFORE BEGINNING:<BR><BR>--ALWAYS use
blocked tips, clean pipettes, and gloved hands when working with RNA.
If you wear a lab coat, DO NOT use the same one used for activities
like seed harvesting or bacterial mini-preps.<BR><BR><BR>
</P>
<P CLASS="western" STYLE="margin-bottom: 0in; border-top: 1px solid #000000; border-bottom: none; border-left: none; border-right: none; padding-top: 0.01in; padding-bottom: 0in; padding-left: 0in; padding-right: 0in">
<B>Reagents and Supplies</B></P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	25ug Total RNA, DNase treated and filtered. 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">1:100 dilution of
	Spike Master Mix. 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Pseudotranscript
		(pmol/ul), in Master Mix <BR>LYS (0.00001) <BR>PHE (0.0001) <BR>THR
		(0.001) <BR>TRP (0.01) <BR>DAP (0.1) 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Pseudotranscript
		(fmol), <B>target usage <I>per 1ug Total RNA</I> </B><BR>LYS
		(0.00002) <BR>PHE (0.0002) <BR>THR (0.002) <BR>TRP (0.02) <BR>DAP
		(0.2) 
		</P>
	</UL>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">SuperScriptIII
	(Invitrogen p/n 18080-051) 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Includes Oligo
		dT(20), dNTP's, buffers, etc. 
		</P>
	</UL>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">Water bath set at
	65C. 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">Air incubator or
	water bath at 50C. 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">Water baths or
	heat blocks set at 85C and 37C. 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">DEPC-treated H2O. 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">RNase free
	tubes</P>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in; border-top: 1px solid #000000; border-bottom: none; border-left: none; border-right: none; padding-top: 0.01in; padding-bottom: 0in; padding-left: 0in; padding-right: 0in">
<BR>
</P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	<B>cDNA Synthesis</B> 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Aliquot 25ug
		Total RNA. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Add: 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">DEPC-treated H<SUB>2</SUB>O
			to 35ul 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">5ul 1:100 spike
			mix dilution 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">5ul 50uM
			Oligo(dT)20 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">5ul 10mM dNTP
			mix 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Incubate 5min at
		65C. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Quick cool, 1
		min. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Add: 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">10ul 10x RT
			Buffer 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">20ul 25mM MgCl<SUB>2</SUB>
						</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">10ul 0.1M DTT 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">5ul RNaseOUT 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">5ul SuperScript
			III Reverse Transcriptase 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Incubate 50min at
		50C. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Incubate 15min at
		85C. Chill. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Add 5ul RNase H. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Incubate 20min at
		37C. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Add
		DEPC-treated H<SUB>2</SUB>O to 3mL (2.895mL). 
		</P>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in; page-break-before: always">
<a name="sec4"></a>
<B>SECTION 4:  cDNA QC TESTING</B></P>
<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
</P>
<P CLASS="western" STYLE="margin-bottom: 0.17in">NOTES BEFORE
BEGINNING:<BR><BR>--ALWAYS wear gloves when touching the optical
plates.<BR><BR>--NEVER write on the optical plates before a
reaction.<BR><BR>--Try not to touch the optical seals in the
center.<BR><BR>--Pay extra attention to seating the seal well,
especially around the edges, with the sealing tool. Neglecting this
can result in leakage during cycling.<BR><BR><BR>
</P>
<P CLASS="western" STYLE="margin-bottom: 0in; border-top: 1px solid #000000; border-bottom: none; border-left: none; border-right: none; padding-top: 0.01in; padding-bottom: 0in; padding-left: 0in; padding-right: 0in">
<B>Supplies and Reagents</B></P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	Finished and diluted cDNA populations from above. 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">2x SYBR Green
	Master mix (ABI p/n 4364346) 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">0.8uM Primer
	Mixes, including: 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">GAPDH 5' End</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">GAPDH 3' End</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">LYS 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">TRP 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">THR 
		</P>
	</UL>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">Optically clear
	384-well plates, barcoded (ABI p/n 4309849) 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Optical seals
	(ABI p/n 4311971)</P>
</UL>
<P CLASS="western" STYLE="margin-left: 0.25in; margin-top: 0.19in; margin-bottom: 0.19in">
<I>* GAPDH primer sequences used came from Michael Udvardi lab.</I></P>
<P CLASS="western" STYLE="margin-bottom: 0in; border-top: 1px solid #000000; border-bottom: none; border-left: none; border-right: none; padding-top: 0.01in; padding-bottom: 0in; padding-left: 0in; padding-right: 0in">
<BR>
</P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	<B>Reaction Setup</B> 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Run one reaction
		with each primer pair for each cDNA population to be checked. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">5ul 2x SYBR Green
		Master Mix, 2.5ul cDNA, 2.5ul 0.8uM primer mix per well. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Seal plate and
		centrifuge 4000rpm, 5min at 4C. 
		</P>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
</P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	<B>Cycling Parameters and document setup</B> 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Open a new
		384-well, 'Absolute quantification' document in SDS. Enter the
		barcode. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Click 'Add
		Detector', bottom middle. 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Select 'SYBR
			Green' and 'Copy to Plate Document' 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Select wells in
		use and click the 'use' box in the setup pane. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Navigate to the
		'Instrument' pane. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Click the end of
		the cycling line, and then click the 'Add Dissociation Stage'
		button. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Alter the volume
		to 10. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Save the
		document. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Click
		'Open/Close', insert the plate in the carrier with well A1 in the
		corner indicated, and click 'Start'. 
		</P>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
</P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	<B>Data Recovery and Analysis</B> 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">When the protocol
		is complete, click the 'Analysis Settings' button (green arrow with
		a blue bar on top) 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Select the
			'Automatic Baseline' radial, and 'Apply'. 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Click 'Analyze'
		(green arrow). 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Switch to the
		'Dissociation Curve' pane, and check for abnormalities. Make notes
		as necessary. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Switch to the
		'Results' pane, and again check for abnormalities. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Using the
		'Export' function under 'File', export the Results Table. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Using EXCEL, or
		similar: 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Calculate
			2^(Ct(GAPDH 5') - Ct(GAPDH 3')) for each cDNA.</P>
			<UL>
				<LI><P CLASS="western" STYLE="margin-bottom: 0in">This value
				should ideally be ~1. 
				</P>
				<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Values above
				2 may indicate poor transcription or degradation of transcripts. 
				</P>
			</UL>
		</UL>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in; page-break-before: always">
<a name="sec5"></a>
<B>SECTION 5:  qPCR REACTION SETUP</B></P>
<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
</P>
<P CLASS="western" STYLE="margin-bottom: 0.17in">NOTES BEFORE
BEGINNING:<BR><BR>--ALWAYS wear gloves when touching the optical
plates.<BR><BR>--NEVER write on the optical plates before a
reaction.<BR><BR>--Try not to touch the optical seals in the
center.<BR><BR>--Pay extra attention to seating the seal well,
especially around the edges, with the sealing tool. Neglecting this
can result in leakage during cycling.<BR><BR><BR>
</P>
<P CLASS="western" STYLE="margin-bottom: 0in; border-top: 1px solid #000000; border-bottom: none; border-left: none; border-right: none; padding-top: 0.01in; padding-bottom: 0in; padding-left: 0in; padding-right: 0in">
<B>Supplies and Reagents</B></P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	Finished and diluted cDNA populations. 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">2x SYBR Green
	Master mix (ABI p/n 4364346) 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">0.8uM Primer
	Mixes, in 384-well plates. 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0in">Optically clear
	384-well plates, barcoded (ABI p/n 4309849) 
	</P>
	<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Optical seals
	(ABI p/n 4311971)</P>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in; border-top: 1px solid #000000; border-bottom: none; border-left: none; border-right: none; padding-top: 0.01in; padding-bottom: 0in; padding-left: 0in; padding-right: 0in">
<BR>
</P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	<B>Reaction Setup</B> 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Run one reaction
		with each primer pair for each cDNA population to be checked. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">PER REACTION: 5ul
		2x SYBR Green Master Mix, 2.5ul cDNA, 2.5ul 0.8uM primer mix per
		well. 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Master mix: 1mL
			cDNA, 2mL 2x SYBR Green Master Mix. 
			</P>
			<UL>
				<LI><P CLASS="western" STYLE="margin-bottom: 0in">Dispense 7.5ul
				per well. 
				</P>
			</UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Add 2.5ul 0.8uM
			primer mix per well, aligning primer plate positions to reaction
			plate positions. 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Seal plate and
		centrifuge 4000rpm, 5min at 4C. 
		</P>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
</P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	<B>Cycling Parameters and document setup</B> 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Open a new
		384-well, 'Absolute quantification' document in SDS. Enter the
		barcode. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Click 'Add
		Detector', bottom middle. 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Select 'SYBR
			Green' and 'Copy to Plate Document' 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Select wells in
		use and click the 'use' box in the setup pane. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Navigate to the
		'Instrument' pane. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Click the end of
		the cycling line, and then click the 'Add Dissociation Stage'
		button. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Alter the cycle
		number to 45, and the volume to 10. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Save the
		document. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Click
		'Open/Close', insert the plate in the carrier with well A1 in the
		corner indicated, and click 'Start'. 
		</P>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
</P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	<B>Data Recovery</B> 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">When the protocol
		is complete, click the 'Analysis Settings' button (green arrow with
		a blue bar on top) 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Select the
			'Automatic Baseline' radial, and 'Apply'. 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Click 'Analyze'
		(green arrow). 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Switch to the
		'Dissociation Curve' pane, and check for. Make notes as necessary. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Switch to the
		'Results' pane, and again check for abnormalities. 
		</P>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Using the
		'Export' function under 'File': 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Export the
			Results Table.</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Export the
			Clipped data. 
			</P>
		</UL>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-left: 1.25in; margin-top: 0.19in; margin-bottom: 0.19in">
<BR><BR>
</P>
<UL>
	<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
	<B>Fluorescence Data and Efficiency Calculations</B> 
	</P>
	<UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">You'll need:
		Excel and LinRegPCR, clipped files. 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Reference for
			LinRegPCR:</P>
		</UL>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-left: 1.25in; margin-top: 0.19in; margin-bottom: 0.19in">
Ramakers C, Ruijter JM, Deprez RH, Moorman AF. (2003).
Assumption-free analysis of quantitative real-time polymerase chain
reaction (PCR) data. <B>Neurosci Lett</B>, <B>339</B>(1): 62-66.</P>
<UL>
	<UL>
		<LI><P CLASS="western" STYLE="margin-top: 0.19in; margin-bottom: 0in">
		Open the clipped file in Excel. 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Delete the first
			range of data. Use the second (Delta Rn). 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Delete <I>contents</I>
			of the column with the header 'Reporter'. Leave the column there. 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Save as Excel
			format. You may put several clipped files in here. 
			</P>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0in">Open LinRegPCR. 
		</P>
		<UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">A dialog box
			will offer to pull data from the <I>active</I> Excel sheet. Please
			make a note of which one you are working with. 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Indicate the
			locations of your data (columns A-AU, rows 2-386). 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Select the
			'Applied Biosystems' radial button and click 'OK'. 
			</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">If you wish to
			change the fit options, select the 'Fit Options' tab. 
			</P>
			<UL>
				<LI><P CLASS="western" STYLE="margin-bottom: 0in">'Number of
				points between 4 and 6 and best correlation coefficient' is
				default and most popular. 
				</P>
				<LI><P CLASS="western" STYLE="margin-bottom: 0in">'Correlation
				Coefficient of at least 0.999 and highest number of points' is
				also acceptable. 
				</P>
			</UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Click 'Fit All'.
						</P>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Review each
			sample; look for anomalies and correct them manually. Examples: 
			</P>
			<UL>
				<LI><P CLASS="western" STYLE="margin-bottom: 0in">Correct: Points
				along the exponential portion of the curve should be selected.
				You are looking at a log plot, so this portion of the curve will
				be linear. 
				</P>
				<LI><P CLASS="western" STYLE="margin-bottom: 0in">Wrong: Points
				along the leveled-off portion (linear) of the curve are selected.
								</P>
				<LI><P CLASS="western" STYLE="margin-bottom: 0in">Wrong: Points
				along the transition area between exponential and linear. 
				</P>
				<LI><P CLASS="western" STYLE="margin-bottom: 0in">Questionable: A
				point at the bottom of the exponential phase, just near the
				'disorganized' points has been selected. This may be a skewed
				point due to proximity to the lower limit of detection or
				background correction range. The author likes to test how the
				values will change if this point is not used. 
				</P>
			</UL>
			<LI><P CLASS="western" STYLE="margin-bottom: 0in">Select 'File &gt;
			Save to Excel'. 
			</P>
			<UL>
				<LI><P CLASS="western" STYLE="margin-bottom: 0in">In the dialog
				box enter 'A' and '1' and click 'ok'. 
				</P>
			</UL>
		</UL>
		<LI><P CLASS="western" STYLE="margin-bottom: 0.19in">Back in Excel,
		change the name of the tab to the plate barcode and your initials.
		Save the file. 
		</P>
	</UL>
</UL>
<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
</P>
<P CLASS="western" STYLE="margin-bottom: 0in"><BR>
</P>
<P CLASS="western" STYLE="margin-bottom: 0in"><B>&gt;&gt;SEE DATA
ANALYSIS DOCUMENTATION FOR MORE INFORMATION&lt;&lt;</B></P>
<DIV TYPE=FOOTER>
	<P STYLE="margin-right: 0.25in; margin-top: 0.71in; margin-bottom: 0in">
	<SPAN ID="Frame1" DIR="LTR" STYLE="position: absolute; top: 0in; left: 0in; width: 0.31in; height: 0.15in; border: none; padding: 0in; background: #ffffff">
		<P STYLE="margin-bottom: 0in"><SDFIELD TYPE=PAGE SUBTYPE=RANDOM FORMAT=PAGE>13</SDFIELD></P>
	</SPAN>The Institute for Genomic Research
	</P>
	<P STYLE="margin-right: 0.25in; margin-bottom: 0in">A division of
	the J. Craig Venter Science Foundation</P>
	<P STYLE="margin-right: 0.25in; margin-bottom: 0in">Chris Town Lab</P>
</DIV>
</BODY>
</HTML>
